src/libstfio/abf/axon/Common/ArrayPtr.hpp

```diff
@@ -14,11 +14,7 @@
 
 #pragma once
 #include <stdlib.h> 
-#if (__cplusplus < 201402L)
-#  include <boost/shared_array.hpp>
-#else
-#  include <memory>
-#endif
+#include <memory>

```

Comment:
The cmake adds `-std=c++11` to build flags so <boost/shared_array.hpp> is not required any more. 

Also `__cplusplus < 201402L` did not work as expected on OSX with `clang` compiler. Probably `__cplusplus` is gcc specifc?